src/main/java/emu/grasscutter/command/commands/NoStaminaCommand.java

```diff
@@ -0,0 +1,39 @@
+package emu.grasscutter.command.commands;
+
+import emu.grasscutter.command.Command;
+import emu.grasscutter.command.CommandHandler;
+import emu.grasscutter.game.player.Player;
+
+import java.util.List;
+
+import static emu.grasscutter.utils.Language.translate;
+
+
+@Command(label = "nostamina", usage = "nostamina [on|off]", permission = "player.nostamina", permissionTargeted = "player.nostamina.others", description = "commands.nostamina.description")
+public final class NoStaminaCommand implements CommandHandler {
+    public static boolean StaminaState = false;
```

Comment:
should be write inside player class instead of the command

Reply:
OK. I‘m busy now.I will fix it.Thanks.

Reply:
I have fixed it.But I eat out now and need to go back and update later.be sorry.